### PR TITLE
Fix: tests

### PR DESCRIPTION
**Pull request by shaiu**
This fixes CI lint + typecheck + format

### CODE REVIEW

File: packages/renderer/.eslintrc.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "env": {
     "browser": true,
-    "node": false
+    "node": true
```
Comment (baruchiro) [inline packages/renderer/.eslintrc.json:4]: We do not accept nodejs code on renderer in this new Vite structure

Comment (shaiu) [reply packages/renderer/.eslintrc.json]: @baruchiro fixed

Comment (baruchiro) [reply packages/renderer/.eslintrc.json]: OK, it is just one thing I saw in quick review. I will review it... I guess... at the end of the week I hope

File: packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx [hunk 2]
```diff
@@ -74,13 +75,7 @@ const EditSheetsExporter: React.FC<EditSheetsExporterProps> = ({
...
       <Card className={styles.card}>
-        <Image
-          className={styles.logo}
-          src={exporterConfig.logo}
-          roundedCircle
-          width={100}
-          height={100}
-        />
+        <Image className={styles.logo} roundedCircle width={100} height={100} />
         <Card.Body className={styles.cardBody}>
```
Comment (baruchiro) [inline packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx:79]: Why did you removed the `src`?

**Review by baruchiro — CHANGES_REQUESTED** [review]
OK I think we almost there, except...

Also, I pushed few changes

File: packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx (hunk 2 above)
Comment (shaiu) [reply packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx:79]: which src?

File: package.json [hunk 3]
```diff
@@ -92,7 +95,9 @@
     "react-bootstrap-table2-editor": "^1.4.0",
     "react-bootstrap-typeahead": "^6.3.2",
     "react-dom": "^18.3.1",
-    "web-vitals": "^4.2.0",
+    "svelte": "^4.2.19",
+    "vite-plugin-node-polyfills": "^0.22.0",
```
[the remark this answers is not in the document]
Comment (shaiu) [reply package.json]: removed

File: packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx (hunk 2 above)
Comment (baruchiro) [reply packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx:79]: The component was:

```
<Image
          className={styles.logo}
          src={exporterConfig.logo}
          roundedCircle
          width={100}
          height={100}
        />
```

With a `src` attribute. It is an image component, without the `src` of the image, it is useless.

Comment (shaiu) [reply packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx:79]: there is no logo under exporterConfig so I guess it's not supposed to work. lint/ts showed this error:
`TS2339: Property 'logo' does not exist on type 'GoogleSheetsConfig'.`

Comment (baruchiro) [reply packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx:79]: And when you run `yarn watch`, do you see any missing icon?

Comment (shaiu) [reply packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx:79]: icons are missing with and without my change

**Pull request merged**